composer.json

```
@@ -27,15 +27,15 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8.0",
-        "illuminate/support": "^5.0|^6.0|^7.0|^8.0|^9.0",
-        "intervention/image": "^2.5",
-        "plank/laravel-mediable": "^2.4|^3.0|^4.2|^5.4"
+        "php": "^8.0|^8.1|^8.2|^8.3",
+        "illuminate/support": "^9.0|^10.0|^11.0",
+        "intervention/image": "^2.5|^3.0",
+        "plank/laravel-mediable": "^5.4|^6.0"
     },
     "require-dev": {
-        "doctrine/dbal": "^2.6",
-        "orchestra/testbench": "^4.8|^5.2|^6.0|^7.0",
-        "phpunit/phpunit": "^9.5",
+        "doctrine/dbal": "^3.2",
+        "orchestra/testbench": "^4.0|^5.0|^6.0|^7.0|^8.0",
```

if we're dropping 6/7/8/9, we can drop matching testbench versions too. 

```suggestion
        "orchestra/testbench": "^7.0|^8.0",
```